test/k8sT/Services.go

```diff
@@ -369,6 +379,7 @@ var _ = Describe("K8sServicesTest", func() {
 						helpers.DefaultNamespace, "test-lb", 30*time.Second)
 					Expect(err).Should(BeNil(), "Cannot retrieve loadbalancer IP for test-lb")
 
+					doRequestsFromOutsideClient("http://"+lbIP, 10)
```

Comment:
Any particular reason for 10 requests? Is this to ensure that the loadbalancing succeeds to all the backends?

Reply:
I copy-pasted this 10 count from the previous tests. I guess for both - making sure that all backends get requested, and checking against any transient failure (which from time to time happens in the NodePort BPF tests).